lib/spack/docs/configuration.rst

```diff
@@ -202,12 +202,12 @@ supply ``-p`` to Spack on the command line, before any subcommands.
 .. _spack-p:
 
 ^^^^^^^^^^^^^^^^^^^
-``spack --profile``
+``spack -p``
 ^^^^^^^^^^^^^^^^^^^
```

Comment:
A simple hack would be to match `spack-[a-z]{2,}`. Would you like me to do that instead, or move them all to `cmd-spack-*`?